explorer/README.md

```diff
@@ -30,29 +30,72 @@ In order to run the explorer locally you need a few things.
 4. GraphQL
 5. UI
 
+
+
+### Devnet
+You can run this all with the following command to build all the backend go binaries and run them. The components in the devnet include the omni network dev net, indexer, db and the graphql server.
+
 ### Fist Time Setup
+You need to have docker installed and running to execute the following command:
 
-You can run this all with the following command to build all the images and run them:
+You can either build all of the component with the following:
+```bash
+make build
+```
 
+then run:
 ```bash
-make run-clean
+make run-devnet
 ```
 
-This will create a fresh build of all the components, run a dev net and start the components in the background. You can access the UI at `http://localhost:3000`, you can access the GraphQL at `http://localhost:8080`.
+or run:
+```bash
+make run-clean
+```
 
-If you want to stop the components you can run:
+### Stopping the network
+If you want to stop the local devnet you can run:
 
 ```bash
 make stop
 ```
 
-### Local Advanced Configurations
+or from the `omni` root folder:
+
+```bash
+make devnet-clean
+```
 
-The explorer needs a copy of the network.json file to be able to know which network to connect to. You can find the network.json file in the `explorer` directory. If you want to run the explorer against a different network you can set the `NETWORK` environment variable to the network you want to run against. For example:
+#### Chains
+
+We currently have 3 chains in the devnet currently, those chainids are:
+- 100
+- 200
+- 16561
+
+So if you were to go and query an xblock, you would need to set one of those as the `sourceChainID` field.
```

Comment:
These docs need to come from the GraphQL schema documentation because GraphQL is self-documented. For example, you might have a resolver `supportedChains` which returns information about them.

Reply:
I'll look into it! For the sake of contractors looking at this tomorrow, I'll add it to my todos, and do a follow up PR